index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+   
+    <div class="start">
+      <header>
+        <a href="#" class="header-title">My Logo</a>
+        <nav>
+          <ul class="hamburger">
+            <li class="stroke"></li>
+            <li class="stroke"></li>
+            <li class="stroke"></li>
+          </ul>
+        </nav>
+      </header>
+      <!-- Homepage-Intro-Section -->
+      <main class="start-intro">
+        <div class="start-intro-container">
+          <div class="start-container">
+            <h1 class="start-intro-heading">I’m Martin Glad to see you!</h1>
+            <p class="start-intro-description">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h3 class="start-intro-connect">LET’S CONNECT</h3>
+            <ul class="social-icons">
+              <li class="icons">
+                <a href="#"><img src="images/Vector (15).png" alt="Twitter" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"
+                  ><img src="images/Linkedin icon.png" alt="LinkedIn"
+                /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="images/Vector (11).png" alt="Medium" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="images/Vector (12).png" alt="GitHub" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"
+                  ><img src="images/Vector (13).png" alt="AngelList"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </main>
```

Comment:
- [ ] Please I highly suggest wrapping the header section with a `section` tag rather than a `main` tag which represents a generic standalone section of the document. This also improves the convention, HTML semantics and user experience. Please check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more information